float64 er
float64 pr
float64[] er_k
float64[] pr_k
float64[] pk_joule
float64[] pk_mech
float64[] pk_indct
float64[] ek_joule
float64[] ek_mech
float64[] ek_indct

string[] iq_jnt_names
